x-pack/test/detection_engine_api_integration/common/config.ts

```diff
@@ -66,7 +66,7 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
         ssl,
         serverArgs: [
           `xpack.license.self_generated.type=${license}`,
-          `xpack.security.enabled=${!disabledPlugins.includes('security') && license === 'trial'}`,
+          `xpack.security.enabled=${!disabledPlugins.includes('security')}`,
```

Comment:
NOTE: This allows us to utilize scenarios outside of trial by including security on everything.